plugins/modules/website.ps1

```diff
@@ -130,6 +151,84 @@ Try {
                 $module.Result.changed = $true
             }
         }
+        # Add Remove or Set bindings if needed
+        if ($bindings) {
```

Comment:
In pwsh an empty hashtable is automatically true when checked like this. If we are wanting to see if add/remove/set is present we would need to do the following to check the entry count as a truthy value where 0 is `$false`.
```suggestion
        if ($bindings.Count) {
```

Reply:
Ah yes because the default module args would be to populate the inner dictionary with the add/set/remove defaults. I think what you've proposed makes sense to me.